packages/ts/frontend/src/Connect.ts

```diff
@@ -16,7 +16,7 @@ import {
 } from './FluxConnection.js';
 import type { VaadinWindow } from './types.js';
 
-const $wnd = window as VaadinWindow;
+const $wnd = self as VaadinWindow;
```

Comment:
I'd suggest using `globalThis` across the code. It's guaranteed to work in any environment